resources/views/dielo.blade.php

```diff
@@ -225,7 +175,8 @@ class="btn btn-default btn-xs btn-outline">{!! $tag !!}</a>
                                 <td class="atribut">{{ trans('dielo.item_attr_technique') }}:</td>
                                 <td>
                                     @foreach ($item->techniques as $technique)
-                                    <a href="{!! URL::to('katalog?technique=' . $technique) !!}">{!! $technique !!}</a><br>
+                                    <a href="{!! URL::to('katalog?technique=' . $technique) !!}">{!! $technique
+                                        !!}</a><br>
```

Comment:
These line breaks look a bit strange (also `{!!` is probably not necessary, but I wouldn't fix that now)
```suggestion
                                    <a href="{!! URL::to('katalog?technique=' . $technique) !!}">{!! $technique !!}</a><br>
```